sumoprojects/sumokoin
# PoW refactoring - part I

Steps to test:
- Create a password protected wallet on the old version
- Compile
- Verify that the client syncs (can be done by temporarily pointing it to a different data dir)
- Sync / change back data dir and verify that the client accepts new blocks after being synced
- Verify that the old wallet is accessible

----

This PR implements structures necessary for supporting two PoW sizes simultaneously without actually changing the underlying PoW
